### DISCUSSION TITLE
Is it possible to use a db function/trigger to save a user's avatar/name after they connect with Google oAuth?

### DISCUSSION THREAD

**Discussion by voicechains** [supabase/supabase#15538] [Questions]
Currently when a new user signs up via Email I save their data to the **public.users** table. 

For users that connect via Google oAuth, I'd like to save their **name** and **avatar** to the **public.users** table. How do I access this data within the database function/trigger below?

![image](https://github.com/supabase/supabase/assets/135489865/3b0b4870-fb12-4584-bc51-f2211ed99d06)

**Comment by GaryAustin1** [comment]
See the SQL option here for an example of that.  https://supabase.com/docs/guides/getting-started/tutorials/with-react#set-up-the-database-schema
